openwisp_users/admin.py

```diff
@@ -189,7 +189,7 @@ def has_change_permission(self, request, obj=None):
         # do not allow operators to edit details of superusers
         # returns 403 if trying to access the change form of a superuser
         if obj and obj.is_superuser and not request.user.is_superuser:
-            return False
+            return False  # pragma: no cover
```

Comment:
the comment should be put one line above as indicated in the [coverage documentation](https://coverage.readthedocs.io/en/coverage-4.3.4/excluding.html):
```python
if obj and obj.is_superuser and not request.user.is_superuser:  # pragma: no cover
    return False
```